docs/governance.rst

```diff
@@ -0,0 +1,253 @@
+- | Enforcer(s) of the `code of conduct`_ `Elizabeth DuPre`_ & `Dan Handwerker`_ & another volunteer
+  |   A person or people someone can go to if they want to report a code of conduct violation
+  |   If this is one person, that person should NOT be on the steering committee
+  |   If this is more than one person, at least one should not be on the steering committee
+  |   Ideal is someone who cares about tedana but DOESN’T know contributors well enough to say, ”Person X would never do that”
+
+
+
+Changing leaders
+````````````````
+Steering committee members can remove themselves from the steering committee at
+any time and open up a call for a new self-nomination. Anyone can request to take
+on a leadership role at any time. Once per year, there should be an explicit call
+to the larger contributor community asking if anyone wants to self nominate for
+membership on the steering committee or other leadership roles. If individuals
+cannot reach consensus on who steps back and who assumes new roles, then a
+majority vote of contributors from the previous 3 years will assign people to
+roles where there are conflicts.
+
+If there are concerns with a tedana steering committee member or leader, any
+enforcer of the code of conduct can ask anyone to step down from a leadership role.
+If a person refuses to step down, then an enforcer of the code of conduct can call
+a vote of contributors to remove an individual from a leadership role in tedana.
+
+
+Decision Making Process
+-----------------------
+
+These rules outlined below are inspired by the 
+`decision-making rules for the BIDS standard <https://github.com/bids-standard/bids-specification/blob/master/DECISION-MAKING.md>`_, which in turn were inspired by the
+`lazy consensus system used in the Apache Foundation <https://www.apache.org/foundation/voting.html>`_,
+and heavily depend on the
+`GitHub Pull Request review system <https://help.github.com/articles/about-pull-requests/>`_.
+
+Definitions
+```````````
+
+Repository
+  `https://github.com/ME-ICA/tedana <https://github.com/ME-ICA/tedana>`_
+
+Contributor
+  Person listed in the `all-contributors file`_.
+  The community decides on the content of this file using the same process as any
+  other change to the Repository (see below) allowing the meaning of "Contributor"
+  to evolve independently of the Decision-making rules.
+
+Maintainer
+  A Contributor responsible for the long term health of the project and the
+  community. Maintainers have additional rights (see Rules) helping them to
+  resolve conflicts and increase the pace of the development when necessary.
+  Any maintainer can self-remove themselves. Any contributor can become a
+  maintainer by request and with the support of the majority of the current
+  maintainers. Current Maintainers:
+
+  +-----------------------------------+-----------------+
```

Comment:
I'm not convinced we should retain this table. It could be aggravating to track.

Reply:
I also worry that time commitments are likely to fluctuate quite a bit over the year, etc... since no one is paid, it's hard to guarantee these times. Both in a good (some weeks I feel like I'm closer to a full hour, or more !) and a bad (other weeks I'm definitely at zero hours) way.
So, I'm +1 on dropping the table.

Reply:
I'm neutral on the table. I just copied it from the design doc. The benefit is it does let new people know that we're all pushing this project forward by doing our small parts. The negative is that the numbers will almost always be wrong. One compromise is set it to a monthly or quarterly commitment rather than a weekly commitment. That is Dan spends 5-10h/quarter

Reply:
That might make the numbers (more or less) correct, but thinking on it more, I'm still worried the table sets the wrong impression (ie, that these folks will work on this for X hours). Most of our development is still done by a volunteer team, and I think it's important prominently acknowledge that.

I'm wondering if this table was inspired from BIDS. They pay their maintainers, so it would make more sense for them.